frontend/appflowy_flutter/lib/workspace/presentation/widgets/float_bubble/question_bubble.dart

```diff
@@ -195,21 +201,31 @@ extension QuestionBubbleExtension on BubbleAction {
         return LocaleKeys.questionBubble_shortcuts.tr();
       case BubbleAction.markdown:
         return LocaleKeys.questionBubble_markdown.tr();
+      case BubbleAction.github:
```

Comment:
Can the text be something like "Report a bug"? Not sure if the length is ok

Reply:
How about Issues?

Reply:
It's a bit techie to me to be honest. If your non-developer friends can understand, it should be good enough.

Reply:
Other options I can think of:
1. Request
2. Feedback
3. Question?